metricbeat/mb/testing/data_generator.go

```diff
@@ -46,40 +47,58 @@ func WriteEvents(f mb.EventsFetcher, t *testing.T) error {
 	if len(events) == 0 {
 		return fmt.Errorf("no events were generated")
 	}
-	return createEvent(events[0], f)
+
+	fullEvent := CreateFullEvent(f, events[0])
+	WriteEventToDataJSON(t, fullEvent)
+	return nil
 }
 
-func createEvent(event common.MapStr, m mb.MetricSet) error {
-	path, err := os.Getwd()
+func CreateFullEvent(ms mb.MetricSet, metricSetData common.MapStr) common.MapStr {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function CreateFullEvent should have comment or be unexported